prod-cockpit-rcar.yaml

```diff
@@ -269,7 +271,9 @@ images:
 parameters:
   # Machines
   MACHINE:
+    desc: "RCAR Gen3-based device"
```

Comment:
Not sure that this change should go in "meta-xt-prod-cockpit-rcar-qt5: add layer to support the qt5." commit